### ISSUE TITLE
Outdated app version

### ISSUE THREAD

**Issue by omyno** [jitsi-contrib/jitsi-helm#8]
The app version has not been updated in 6 months. Is there a plan to keep up with the released docker images?

If not, is it safe to manually bump up the version?

**Comment by omyno** [comment]
For the time being, I updated all docker image versions like this:

```
helm -n jitsi-meet template jitsi-meet jitsi/jitsi-meet -f jitsi.values.yaml > jitsi-meet.template.yaml
vim jitsi-meet.template.yaml
kubectl -n jitsi-meet apply -f jitsi-meet.template.yaml
```

This worked fine.

**Comment by sapkra** [comment]
Should be fixed with #10.
